src/crate/client/doctests/cursor.txt

```diff
@@ -296,6 +304,69 @@ closed connection an ``ProgrammingError`` exception will be raised::
     ...
     crate.client.exceptions.ProgrammingError: Cursor closed
 
+
+Python data type conversion
+===========================
+
+The cursor object can optionally convert database types to native Python data
+types. Currently, this is implemented for the CrateDB data types ``IP`` and
+``TIMESTAMP`` on behalf of the ``DefaultTypeConverter``.
+
+::
+
+    >>> cursor = connection.cursor(converter=Cursor.get_default_converter())
+
+    >>> connection.client.set_next_response({
+    ...     "col_types": [4, 5, 11],
+    ...     "rows":[ [ "foo", "10.10.10.1", 1658167836758 ] ],
+    ...     "cols":[ "name", "address", "timestamp" ],
+    ...     "rowcount":1,
+    ...     "duration":123
+    ... })
+
+    >>> cursor.execute('')
```

Comment:
I think this would confuse users because this is a mocked cursor and `client.set_next_response` isn't something that they can use.

Either we need to point this out explicitly, hide it or use real data and don't mock it.

Reply:
At the beginning of the file `cursor.txt`, there is a corresponding note about this detail.

> Hardcode the next response of the mocked connection client, so we won't need a sql statement
to execute.

After that, the `set_next_response()` method is already used five times within `cursor.txt`, so I didn't hesitate to continue implementing the test cases this way.

Reply:
Is there a reason we can't/shouldn't hide it for the rendered docs?

Reply:
As far as I can see, `cursor.txt` is not part of the rendered docs at https://crate.io/docs/python/, it is only used as a doctest.

Reply:
I've created #448 to have a separate discussion about the topic how to improve/adjust doctests vs. rendered documentation.

Reply:
I totally agree with you. I think your guidance should be implemented seperately, being tracked by #448. If you want to see it sooner than later, than I will be happy to tackle it before coming back to this patch.

Reply:
Sounds good to handle this separately in a follow up